app/routes/index.tsx

```diff
@@ -54,19 +64,37 @@ export default function Index(): JSX.Element {
 	return (
 		<>
 			<Navbar groupedPages={groupedPages} />
-			<Container w="full" maxW="7xl" mx="auto" py={8} centerContent>
-				<chakra.section>
+			<Container
+				w="full"
+				// bg="#f6f9fc"
+				bg={useColorModeValue("gray.50", "")}
```

Comment:
I'd avoid setting custom colors anywhere if possible, as it goes against chakra's style guide. This doesn't seem to account for dark mode either?